org.scummvm.ScummVM.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id": "org.scummvm.ScummVM",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
```

Comment:
Does it need network access?

Reply:
Yes, it has a cloud feature:
![scummvm_network](https://user-images.githubusercontent.com/2339930/41205756-d018bf48-6cf9-11e8-83aa-89bdf515a739.png).